app/styles/auk/_auk-additions.scss

```diff
@@ -140,3 +140,9 @@ table tr.lt-expanded-row {
 .vlc-procedure-step--nopadding {
   padding: 0;
 }
+
+$vl-picton-blue: #16465B;
```

Comment:
@brenner-company deze heb jij in #1329 verwijderd, en als ik het goed begrepen heb ben je daar overgestapt naar `<AuPill @skin="ongoing">`. Ik heb eerst geprobeerd om dit hier ook te doen door `AccessLevelPill` aan te passen om een `<AuPill>` te gebruiken, maar dan kreeg ik de iconen niet werkend (het probeert de iconen van volgend URL te fetchen, wat niet bestaat: `/@appuniversum/ember-appuniversum/appuniversum-symbolset.svg#ministerraad`).

Ik heb de changes daarom in de meantime teruggezet, maar mocht er een betere oplossing zijn wil ik dit zeker aanpasen

Reply:
@sergiofenoll Dit zal komen door het feit dat die specifieke icons (nog) niet beschikbaar zijn binnen `ember-appuniversum` (zie https://appuniversum.github.io/ember-appuniversum/?path=/story/components-brand-auicon--icons) dus dit mag je inderdaad zo even laten denk ik.

@Dietr Is er binnen `ember-appuniversum` de mogelijkheid om deze icons (zie https://github.com/kanselarij-vlaanderen/au-kaleidos-icons/pull/5) toe voegen of is dit iets te specifiek?

Reply:
@brenner-company Kan er zeker bij, zal die straks toevoegen

Reply:
@Dietr Super, merci! Houden jij/jullie dan in het geval van appuniversum de naam van de icons ook zo generic mogelijk? Bv. [intern-regering.svg](https://github.com/kanselarij-vlaanderen/au-kaleidos-icons/blob/f0931f44d45e78bbd4e810afbe78ec4d66e17de1/svg/intern-regering.svg) is dan iets als `two-of-four-persons` of is dat wat overbodig?

Reply:
Zit in de nieuwe release: https://github.com/appuniversum/ember-appuniversum/releases/tag/v1.0.16
Heb de namen van de icons ook aangepast: https://appuniversum.github.io/ember-appuniversum/?path=/story/components-brand-auicon--icons

Reply:
Bedankt!